client/guidestours/index.js

```diff
@@ -50,6 +62,96 @@ const bullseyePositioners = {
 };
 
 class GuidesStep extends Component {
+	getStepPosition() {
+		const { placement = 'center', target } = this.props;
+		const rect = target
+			? target.getBoundingClientRect()
+			: global.window.document.body.getBoundingClientRect();
+
+		return dialogPositioners[ placement ]( rect )
+	}
+
+	render() {
+		const stepPos = this.getStepPosition();
+		const stepCoords = posToCss( stepPos );
+
+		const { text, onNext, onQuit } = this.props;
+		return (
+			<Card className="guidestours__step" style={ stepCoords } >
+				<p>{ text }</p>
+				<div className="guidestours__buttonrow__choice">
+					<Button onClick={ onNext } primary>Continue</Button>
+					<Button onClick={ onQuit } borderless>Do this later.</Button>
+				</div>
+			</Card>
+		);
+	}
+}
+
+class GuidesFirstStep extends GuidesStep {
```

Comment:
Same. I'd go for the separate module too, but even in the short term — before actually creating a new file for said module — I'd make sure any common functionality/logic can be soundly abstracted away into pure functions in the file.